src/bzflag/DisplayMenu.cxx

```diff
@@ -404,10 +393,6 @@ void            DisplayMenu::callback(HUDuiControl* w, const void* data)
         BZDB.set("dither", list->getIndex() ? "1" : "0");
         sceneRenderer->notifyStyleChange();
         break;
-    case '2':
```

Comment:
you shouldn't just remove case 2. Sadly, there is no default case in this switch. Either add a default case (which is best) or an empty or error case for 2.

Reply:
I agree! Doing it